### PR TITLE
Verify ssl

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
Includes 3 commits:
- add an option for certificates's validation. needed to work with self-signed certificates
- change string and quotes format according to https://github.com/ansible/ansible-language-server
- move vars file to the right place.

### CODE REVIEW

File: roles/setup/defaults/main.yml [hunk 1]
```diff
@@ -1,19 +1,18 @@
 admin_url: http://app:8000/admin
 auth_client_id: admin-cli
-auth_keycloak_url: '{{ lookup(''env'', ''PINAKES_KEYCLOAK_URL'') | default(''http://default_keycloak:8080/auth'',
-  True) }}'
-auth_password: '{{ lookup(''env'', ''PINAKES_KEYCLOAK_PASSWORD'') | default(''admin'', True)
-  }}'
-auth_realm: '{{ lookup(''env'', ''PINAKES_KEYCLOAK_MASTER_REALM'') | default(''master'', True) }}'
-auth_username: '{{ lookup(''env'', ''PINAKES_KEYCLOAK_USER'') | default(''admin'', True) }}'
+auth_keycloak_url: "{{ lookup('env', 'PINAKES_KEYCLOAK_URL') | default('http://default_keycloak:8080/auth', True) }}"
```
Comment (cutwater) [inline roles/setup/defaults/main.yml:3]: @Alex-Izquierdo Generally speaking is it advisable for role to use environment variables directly? Or should the role be configured by the playbook, with appropriate environment variables?

Comment (Alex-Izquierdo) [reply roles/setup/defaults/main.yml:3]: I think @mkanoor did it in that way to take the values from the dev environments. As long as have default values I don't see any problem.

File: roles/setup/tasks/create_policy.yml [hunk 2]
```diff
@@ -1,68 +1,68 @@
...
-    - 'create_policy.json is defined'
-    - 'create_policy.json.id is defined'
-- name: 'Get policy if we didn''t create a new policy'
+    - "create_policy.status == 201"
+    - "create_policy.json is defined"
+    - "create_policy.json.id is defined"
+- name: "Get policy if we didn't create a new policy"
   register: get_policy_list
   uri:
+    validate_certs: "{{ verify_ssl }}"
```
Comment (mkanoor) [inline roles/setup/tasks/create_policy.yml]: @Alex-Izquierdo Should the verify_ssl be called verify_keycloak_ssl?

File: roles/setup/tasks/main.yml [hunk 3]
```diff
@@ -1,72 +1,74 @@
...
-      frontendUrl: '{{ realm_frontend_url }}'
-    id: '{{ realm_name }}'
-    realm: '{{ realm_name }}'
+      frontendUrl: "{{ realm_frontend_url }}"
+    id: "{{ realm_name }}"
+    realm: "{{ realm_name }}"
     state: present
-    validate_certs: true
-  name: 'Create or update AAP Keycloak realm'
+    validate_certs: "{{ verify_ssl }}"
```
Comment (mkanoor) [inline roles/setup/tasks/main.yml]: @Alex-Izquierdo According to the ansible module for keycloak this only supports boolean data. https://docs.ansible.com/ansible/latest/collections/community/general/keycloak_client_module.html#parameter-validate_certs

**Comment by mkanoor** [comment]
@Alex-Izquierdo There is an ansible linter you might want to pass it thru that. https://ansible-lint.readthedocs.io/en/latest/

File: roles/setup/tasks/main.yml (hunk 3 above)
Comment (Alex-Izquierdo) [reply roles/setup/tasks/main.yml]: I don't understand this comment. The variable takes already a boolean.

File: roles/setup/tasks/create_policy.yml (hunk 2 above)
Comment (Alex-Izquierdo) [reply roles/setup/tasks/create_policy.yml]: changed

**Comment by Alex-Izquierdo** [comment]
> 

@mkanoor I work with vscode and the official extension which relies on https://github.com/ansible/ansible-language-server and ansible-lint.

**Review by mkanoor — APPROVED** [review]

**Pull request merged**